osu.Game/Graphics/UserInterface/AccuracyCounter.cs

```diff
@@ -0,0 +1,109 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Transformations;
+using osu.Framework.Timing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    /// <summary>
+    /// Used as an accuracy counter. Represented visually as a percentage, internally as a fraction.
+    /// </summary>
+    public class AccuracyCounter : RollingCounter<float>
+    {
+        protected override Type transformType => typeof(TransformAccuracy);
+
+        private long numerator = 0;
+        public long Numerator
+        {
+            get
+            {
+                return numerator;
+            }
+            set
+            {
+                numerator = value;
+                updateCount();
+            }
+        }
+
+        private ulong denominator = 0;
+        public ulong Denominator
+        {
+            get
+            {
+                return denominator;
+            }
+            set
+            {
+                denominator = value;
+                updateCount();
+            }
+        }
+
+        public AccuracyCounter()
+        {
+            RollingDuration = 500;
+            RollingEasing = EasingTypes.Out;
+        }
+
+        public override void Load(BaseGame game)
+        {
+            base.Load(game);
+
+            updateCount();
+            StopRolling();
+        }
+
+        public void SetCount(long num, ulong den)

```

Comment:
I'd probably have a standardised method of setting the current value at the base RollingCounter class (seems you already do actually, so this shouldn't even be present).